api/serializers.py

```diff
@@ -172,6 +172,15 @@ class Meta:
 
     labels = BaseLabelSerializer(many=True, read_only=True)
 
+    created_by = UserSerializer(read_only=True)
+    created_by_id = serializers.PrimaryKeyRelatedField(
```

Comment:
The create function requires the context to have the logged in user. Instead, In my new commit, I enforce the user inside the view class.